Material.Blazor/Foundation/ComponentFoundation.cs

```diff
@@ -137,10 +137,10 @@ protected virtual async ValueTask DisposeAsync(bool disposing)
                 return;
             }
 
-            if (HasInstantiated)
-            {
-                await DestroyMcwComponent();
-            }
+            //if (HasInstantiated)
+            //{
+            //    await DestroyMcwComponent();
```

Comment:
We should consider marking `DestroyMcwComponent` as obsolete to make clean up easier

Reply:
Good idea. Doing that now.

Reply:
Better still I've just deleted all `DestroyMcwComponent`s from the C# code. I haven't yet looked at removing the JS code. I'm in two minds and am OK with leaving it around for a while.